src/lib.rs

```diff
@@ -121,7 +121,7 @@
 #![warn(missing_docs)]
 #![warn(unused_qualifications)]
 #![deny(unreachable_pub)]
-#![deny(deprecated)]
+//#![deny(deprecated)]
```

Comment:
Please annotate the individual tests/use sites with `#[allow(deprecated)]` instead. That way, one can easily tally all the deprecated code with `grep`.

Reply:
Great to learn this technique!